"build -j2 native native" results in "System is unknown to cmake"

```
heidezomp@acerola:~/Downloads/zig-bootstrap-0.6.0$ ./build -j2 native native
[... compiling llvm/zig natively works fine ...]
[100%] Built target zig
Install the project...
-- Install configuration: "Release"
-- Installing: /home/heidezomp/Downloads/zig-bootstrap-0.6.0/out/host/bin/zig
-- Set runtime path of "/home/heidezomp/Downloads/zig-bootstrap-0.6.0/out/host/bin/zig" to ""
-- Installing: /home/heidezomp/Downloads/zig-bootstrap-0.6.0/out/host/lib
-- The C compiler identification is Clang 10.0.0
-- The CXX compiler identification is Clang 10.0.0
-- The ASM compiler identification is Clang
-- Found assembler: /home/heidezomp/Downloads/zig-bootstrap-0.6.0/out/host/bin/zig
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Check for working C compiler: /home/heidezomp/Downloads/zig-bootstrap-0.6.0/out/host/bin/zig
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Check for working C compiler: /home/heidezomp/Downloads/zig-bootstrap-0.6.0/out/host/bin/zig -- works
-- Detecting C compiler ABI info
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Detecting C compiler ABI info - done
-- Detecting C compile features
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Detecting C compile features - done
-- Check for working CXX compiler: /home/heidezomp/Downloads/zig-bootstrap-0.6.0/out/host/bin/zig
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Check for working CXX compiler: /home/heidezomp/Downloads/zig-bootstrap-0.6.0/out/host/bin/zig -- works
-- Detecting CXX compiler ABI info
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Detecting CXX compile features - done
-- clang project is enabled
-- clang-tools-extra project is disabled
-- compiler-rt project is disabled
-- debuginfo-tests project is disabled
-- libc project is disabled
-- libclc project is disabled
-- libcxx project is disabled
-- libcxxabi project is disabled
-- libunwind project is disabled
-- lld project is enabled
-- lldb project is disabled
-- llgo project is disabled
-- mlir project is disabled
-- openmp project is disabled
-- parallel-libs project is disabled
-- polly project is disabled
-- pstl project is disabled
-- Could NOT find Z3: Found unsuitable version "0.0.0", but required is at least "4.7.1" (found Z3_LIBRARIES-NOTFOUND)
-- Performing Test LLVM_LIBSTDCXX_MIN
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test LLVM_LIBSTDCXX_MIN - Success
-- Performing Test LLVM_LIBSTDCXX_SOFT_ERROR
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test LLVM_LIBSTDCXX_SOFT_ERROR - Success
-- Looking for dlfcn.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for dlfcn.h - found
-- Looking for errno.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for errno.h - found
-- Looking for fcntl.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for fcntl.h - found
-- Looking for link.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for link.h - found
-- Looking for malloc/malloc.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for malloc/malloc.h - not found
-- Looking for pthread.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for pthread.h - found
-- Looking for signal.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for signal.h - found
-- Looking for sys/ioctl.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for sys/ioctl.h - found
-- Looking for sys/mman.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for sys/mman.h - found
-- Looking for sys/param.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for sys/param.h - found
-- Looking for sys/resource.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for sys/resource.h - found
-- Looking for sys/stat.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for sys/stat.h - found
-- Looking for sys/time.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for sys/time.h - found
-- Looking for sys/types.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for sys/types.h - found
-- Looking for termios.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for termios.h - found
-- Looking for unistd.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for unistd.h - found
-- Looking for valgrind/valgrind.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for valgrind/valgrind.h - not found
-- Looking for zlib.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for zlib.h - found
-- Looking for fenv.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for fenv.h - found
-- Looking for FE_ALL_EXCEPT
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for FE_ALL_EXCEPT - found
-- Looking for FE_INEXACT
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for FE_INEXACT - found
-- Looking for mach/mach.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for mach/mach.h - not found
-- Looking for histedit.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for histedit.h - not found
-- Looking for CrashReporterClient.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for CrashReporterClient.h - not found
-- Looking for pthread_create in pthread
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for pthread_create in pthread - found
-- Looking for pthread_getspecific in pthread
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for pthread_getspecific in pthread - found
-- Looking for pthread_rwlock_init in pthread
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for pthread_rwlock_init in pthread - found
-- Looking for pthread_mutex_lock in pthread
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for pthread_mutex_lock in pthread - found
-- Looking for dlopen in dl
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for dlopen in dl - found
-- Looking for clock_gettime in rt
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for clock_gettime in rt - found
-- Looking for pfm_initialize in pfm
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for pfm_initialize in pfm - not found
-- Looking for pthread.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for pthread.h - found
-- Looking for pthread_create
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for pthread_create - found
-- Found Threads: TRUE  
-- Looking for compress2 in z
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for compress2 in z - found
-- Looking for setupterm in terminfo
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for setupterm in terminfo - not found
-- Looking for setupterm in tinfo
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for setupterm in tinfo - found
-- Looking for xar_open in xar
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for xar_open in xar - not found
-- Looking for arc4random
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for arc4random - not found
-- Looking for backtrace
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for backtrace - found
-- backtrace facility detected in default set of libraries
-- Could NOT find Backtrace (missing: Backtrace_INCLUDE_DIR) 
-- Performing Test C_SUPPORTS_WERROR_UNGUARDED_AVAILABILITY_NEW
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test C_SUPPORTS_WERROR_UNGUARDED_AVAILABILITY_NEW - Success
-- Looking for _Unwind_Backtrace
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for _Unwind_Backtrace - found
-- Looking for getpagesize
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for getpagesize - found
-- Looking for sysconf
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for sysconf - found
-- Looking for getrusage
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for getrusage - found
-- Looking for setrlimit
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for setrlimit - found
-- Looking for isatty
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for isatty - found
-- Looking for futimens
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for futimens - found
-- Looking for futimes
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for futimes - found
-- Looking for posix_fallocate
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for posix_fallocate - found
-- Looking for sigaltstack
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for sigaltstack - found
-- Looking for lseek64
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for lseek64 - found
-- Looking for mallctl
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for mallctl - not found
-- Looking for mallinfo
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for mallinfo - found
-- Looking for malloc_zone_statistics
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for malloc_zone_statistics - not found
-- Looking for getrlimit
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for getrlimit - found
-- Looking for posix_spawn
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for posix_spawn - found
-- Looking for pread
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for pread - found
-- Looking for sbrk
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for sbrk - found
-- Looking for strerror
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for strerror - found
-- Looking for strerror_r
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for strerror_r - found
-- Looking for strerror_s
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for strerror_s - not found
-- Looking for setenv
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for setenv - found
-- Looking for dlopen
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for dlopen - found
-- Looking for dladdr
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for dladdr - not found
-- Performing Test HAVE_STRUCT_STAT_ST_MTIMESPEC_TV_NSEC
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test HAVE_STRUCT_STAT_ST_MTIMESPEC_TV_NSEC - Failed
-- Performing Test HAVE_STRUCT_STAT_ST_MTIM_TV_NSEC
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test HAVE_STRUCT_STAT_ST_MTIM_TV_NSEC - Success
-- Looking for __GLIBC__
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for __GLIBC__ - found
-- Looking for sched_getaffinity
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for sched_getaffinity - found
-- Looking for CPU_COUNT
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for CPU_COUNT - found
-- Looking for pthread_getname_np
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for pthread_getname_np - found
-- Looking for pthread_setname_np
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for pthread_setname_np - found
-- Performing Test HAVE_STD_IS_TRIVIALLY_COPYABLE
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test HAVE_STD_IS_TRIVIALLY_COPYABLE - Success
-- Performing Test HAVE_CXX_ATOMICS_WITHOUT_LIB
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test HAVE_CXX_ATOMICS_WITHOUT_LIB - Success
-- Performing Test HAVE_CXX_ATOMICS64_WITHOUT_LIB
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test HAVE_CXX_ATOMICS64_WITHOUT_LIB - Success
-- Performing Test LLVM_HAS_ATOMICS
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test LLVM_HAS_ATOMICS - Success
-- Performing Test SUPPORTS_VARIADIC_MACROS_FLAG
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test SUPPORTS_VARIADIC_MACROS_FLAG - Success
-- Performing Test SUPPORTS_GNU_ZERO_VARIADIC_MACRO_ARGUMENTS_FLAG
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test SUPPORTS_GNU_ZERO_VARIADIC_MACRO_ARGUMENTS_FLAG - Success
-- Native target architecture is X86
-- Threads enabled.
-- Doxygen disabled.
-- Go bindings disabled.
-- Could NOT find OCaml (missing: OCAMLFIND OCAML_VERSION OCAML_STDLIB_PATH) 
-- OCaml bindings disabled.
-- Could NOT find Python module pygments
-- Could NOT find Python module pygments.lexers.c_cpp
-- Could NOT find Python module yaml
-- LLVM host triple: x86_64-unknown-linux-gnu
-- LLVM default target triple: native
CMake Error at cmake/modules/HandleLLVMOptions.cmake:120 (MESSAGE):
  Unable to determine platform
Call Stack (most recent call first):
  CMakeLists.txt:648 (include)


-- Performing Test C_SUPPORTS_FPIC
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test C_SUPPORTS_FPIC - Success
-- Performing Test CXX_SUPPORTS_FPIC
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test CXX_SUPPORTS_FPIC - Success
-- Building with -fPIC
-- Performing Test SUPPORTS_FVISIBILITY_INLINES_HIDDEN_FLAG
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test SUPPORTS_FVISIBILITY_INLINES_HIDDEN_FLAG - Success
-- Performing Test C_SUPPORTS_WERROR_DATE_TIME
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test C_SUPPORTS_WERROR_DATE_TIME - Success
-- Performing Test CXX_SUPPORTS_WERROR_DATE_TIME
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test CXX_SUPPORTS_WERROR_DATE_TIME - Success
-- Performing Test CXX_SUPPORTS_WERROR_UNGUARDED_AVAILABILITY_NEW
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test CXX_SUPPORTS_WERROR_UNGUARDED_AVAILABILITY_NEW - Success
-- Performing Test CXX_SUPPORTS_MISSING_FIELD_INITIALIZERS_FLAG
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test CXX_SUPPORTS_MISSING_FIELD_INITIALIZERS_FLAG - Success
-- Performing Test C_SUPPORTS_IMPLICIT_FALLTHROUGH_FLAG
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test C_SUPPORTS_IMPLICIT_FALLTHROUGH_FLAG - Success
-- Performing Test CXX_SUPPORTS_IMPLICIT_FALLTHROUGH_FLAG
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test CXX_SUPPORTS_IMPLICIT_FALLTHROUGH_FLAG - Success
-- Performing Test C_SUPPORTS_COVERED_SWITCH_DEFAULT_FLAG
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test C_SUPPORTS_COVERED_SWITCH_DEFAULT_FLAG - Success
-- Performing Test CXX_SUPPORTS_COVERED_SWITCH_DEFAULT_FLAG
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test CXX_SUPPORTS_COVERED_SWITCH_DEFAULT_FLAG - Success
-- Performing Test CXX_SUPPORTS_CLASS_MEMACCESS_FLAG
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test CXX_SUPPORTS_CLASS_MEMACCESS_FLAG - Failed
-- Performing Test CXX_SUPPORTS_NOEXCEPT_TYPE_FLAG
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test CXX_SUPPORTS_NOEXCEPT_TYPE_FLAG - Success
-- Performing Test CXX_WONT_WARN_ON_FINAL_NONVIRTUALDTOR
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test CXX_WONT_WARN_ON_FINAL_NONVIRTUALDTOR - Success
-- Performing Test C_SUPPORTS_DELETE_NON_VIRTUAL_DTOR_FLAG
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test C_SUPPORTS_DELETE_NON_VIRTUAL_DTOR_FLAG - Success
-- Performing Test CXX_SUPPORTS_DELETE_NON_VIRTUAL_DTOR_FLAG
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test CXX_SUPPORTS_DELETE_NON_VIRTUAL_DTOR_FLAG - Success
-- Performing Test C_WCOMMENT_ALLOWS_LINE_WRAP
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test C_WCOMMENT_ALLOWS_LINE_WRAP - Success
-- Performing Test C_SUPPORTS_STRING_CONVERSION_FLAG
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test C_SUPPORTS_STRING_CONVERSION_FLAG - Success
-- Performing Test CXX_SUPPORTS_STRING_CONVERSION_FLAG
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test CXX_SUPPORTS_STRING_CONVERSION_FLAG - Success
-- Performing Test C_SUPPORTS_FNO_FUNCTION_SECTIONS
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test C_SUPPORTS_FNO_FUNCTION_SECTIONS - Success
-- Performing Test C_SUPPORTS_FFUNCTION_SECTIONS
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test C_SUPPORTS_FFUNCTION_SECTIONS - Success
-- Performing Test CXX_SUPPORTS_FFUNCTION_SECTIONS
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test CXX_SUPPORTS_FFUNCTION_SECTIONS - Success
-- Performing Test C_SUPPORTS_FDATA_SECTIONS
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test C_SUPPORTS_FDATA_SECTIONS - Success
-- Performing Test CXX_SUPPORTS_FDATA_SECTIONS
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test CXX_SUPPORTS_FDATA_SECTIONS - Success
-- Looking for os_signpost_interval_begin
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for os_signpost_interval_begin - not found
-- Found PythonInterp: /usr/bin/python3.6 (found version "3.6.9") 
-- Constructing LLVMBuild project information
-- Found Git: /usr/bin/git (found version "2.17.1") 
-- Linker detection: unknown
-- Setting native build dir to /home/heidezomp/Downloads/zig-bootstrap-0.6.0/out/build-llvm-native/NATIVE
-- Linker detection: unknown
-- Targeting AArch64
-- Targeting AMDGPU
-- Targeting ARM
-- Targeting BPF
-- Targeting Hexagon
-- Targeting Lanai
-- Targeting Mips
-- Targeting MSP430
-- Targeting NVPTX
-- Targeting PowerPC
-- Targeting RISCV
-- Targeting Sparc
-- Targeting SystemZ
-- Targeting WebAssembly
-- Targeting X86
-- Targeting XCore
-- Targeting AVR
-- Looking for sys/resource.h
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Looking for sys/resource.h - found
-- Clang version: 10.0.0
-- Performing Test CXX_SUPPORTS_NO_NESTED_ANON_TYPES_FLAG
System is unknown to cmake, create:
Platform/Native to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
-- Performing Test CXX_SUPPORTS_NO_NESTED_ANON_TYPES_FLAG - Success
-- LLD version: 10.0.0
-- Configuring incomplete, errors occurred!
See also "/home/heidezomp/Downloads/zig-bootstrap-0.6.0/out/build-llvm-native/CMakeFiles/CMakeOutput.log".
See also "/home/heidezomp/Downloads/zig-bootstrap-0.6.0/out/build-llvm-native/CMakeFiles/CMakeError.log".
heidezomp@acerola:~/Downloads/zig-bootstrap-0.6.0$ 
```

My cmake is version 3.10.2.

--- a/README.md
+++ b/README.md
@@ -21,14 +21,15 @@ This repository copies sources from upstream, with no patches.
 ## Build Instructions
 
 ```
-build -j1 native baseline
+build -j1 <arch>-<os>-<abi> baseline
 ```
 
 All parameters are required:
 
  * `-j1`: Replace with your jobs parameter to make.
- * `native`: Replace with one of the Supported Triples below, or use `native`
-   for the native triple.
+ * `<arch>-<os>-<abi>`: Replace with one of the Supported Triples below, or use
+   `native` for the `<arch>` value (e.g. `native-linux-gnu`) to use the native
+    architecture.
  * `baseline`: Replace with a `-mcpu` parameter of Zig. `baseline` means
    it will target a generic CPU for the target. `native` means it will target
    the native CPU. See the Zig documentation for more details.

